src/components/ToastContainer.tsx

```diff
@@ -4,7 +4,19 @@ import { Toast } from './Toast';
 import { zIndex } from '../../src/theme';
 import { ToastData } from '../../src/types';
 
-const StyledToastContainer = styled(BodyPortal)`
+const StyledToastContainer = styled.div`
+  ${(props: {inline: boolean}) => !props.inline && css`
+    position: fixed;
+    right: 2rem;
+  `}
+  z-index: ${zIndex.toasts};
+  display: grid;
+  justify-items: center;
+  justify-content: center;
+  gap: 1vh;
```

Comment:
This looks like the same css, can we put it into a variable for both?

Reply:
I thought about extracting the template string/array and reusing it, but I ran into issues with the types, because I think styled-components uses a generic type that gets some of its values from the template string/array.

Reply:
Oh, I can probably make a factory function